scripts/fonts.sh

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
 
+# https://github.com/cfpb/design-system/pull/2309/files#diff-90f15467c133b5bbfee15918b85af670f342d74323094f8caad8fbb2b2628878
+FONT_VARIABLE="source-sans-3-latin-wght-normal.woff2"
+BASE_DIR="./static/fonts"
+
 # Add required webfonts locally to src/static/fonts/ directory.
 cd src
 mkdir ./static/
-mkdir ./static/fonts
-cd ./static/fonts
-prefix="https://github.com/cfpb/consumerfinance.gov/blob/main/static.in/cfgov-fonts/fonts/"
-fonts=("2cd55546-ec00-4af9-aeca-4a3cd186da53.woff2" "627fbb5a-3bae-4cd9-b617-2f923e29d55e.woff2")
-for font in ${fonts[@]}; do
-  # Flags: L = follow redirects.
-  curl -L "${prefix}${font}?raw=true" > ${font}
-done
-cd ../../../
+# Add required webfonts locally to docs/fonts/ directory.
+mkdir $BASE_DIR
+echo "Copying ./node_modules/@fontsource-variable/source-sans-3/files/$FONT_VARIABLE to $BASE_DIR/$FONT_VARIABLE"
+cp ./node_modules/@fontsource-variable/source-sans-3/files/$FONT_VARIABLE $BASE_DIR/$FONT_VARIABLE
```

Comment:
This script should copy the packaged font file to the location where it will be served. This can either be done by install the font file as a dependency here via package.json (`yarn add @fontsource-variable/source-sans-3`), or we could deliver the font file through design system npm package (since it is already installed there). What do you think would be nicer, from a dev experience? Either way, you have to reach into `node_modules` to get the font file.